amica/src/components/vrmViewer.tsx

```
@@ -35,45 +57,71 @@ export default function VrmViewer({ chatMode }: { chatMode: boolean }) {
           try {
+            console.log('[VrmViewer] Setting up viewer...');
+            await viewer.setup(canvas);
+
+            console.log('[VrmViewer] Loading scenario from:', config('scenario_url'));
             await viewer.scenario.loadScenario(config('scenario_url'), viewer, globalHookManager);
             resolve(true);
           } catch (e) {
+            console.error('[VrmViewer] Setup or scenario loading failed:', e);
             reject(e);
           }
         })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)